.github/workflows/linter-analysis.yaml

```diff
@@ -1,7 +1,7 @@
 name: Linter Analysis
 on:
   push:
-    branches: [ '*' ] # '*' will cause the workflow to run on all commits to all branches.
+    branches: [ '**' ] # '**' will cause the workflow to run on all commits to all branches, including those with path separators
```

Comment:
Guessing this is based on the new foo/bar-branch naming convention?

Reply:
That's correct, changing it to `branches: [ '**' ]` allows for the workflow to be automatically run again `foo/bar-branch` naming convention as well those without forward slashes , `foo-bar-branch`